src/app/component/shared/checkbox/checkbox.component.ts

```diff
@@ -21,12 +21,13 @@ import {
       </div>
     </div>
   `,
-  changeDetection: ChangeDetectionStrategy.OnPush
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class CheckboxComponent implements OnInit {
   @Input() id: string;
   @Input() checked: boolean;
   @Input() disabled: boolean;
+  @Input() ngClass: { [key: string]: boolean } = { "mx-auto": true };
```

Comment:
I'd prefer we didn't let class names leak into components. Can we do the fallback in the template rather than as a default  parameter here?